### ISSUE TITLE
MotionPath Example

### ISSUE THREAD

**Issue by karneaud** [legomushroom/mojs#40]
Here is a [demo](https://jsfiddle.net/karneaud/gvzekze5/63/) using mojs.MotionPath to move an element along an SVG Path.

**Comment by legomushroom** [comment]
nice! so you are using a helper to recalculate coordinates to length. We might add this to motion path module by default. What to do when you have multiple equal coordinates? Use the first one?

**Comment by karneaud** [comment]
@legomushroom yeah....not like each one of the coords is going to be a different outcome anyways
